tests/com/github/bazelbuild/rules_jvm_external/JarUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.github.bazelbuild.rules_jvm_external;
+
+import com.github.bazelbuild.rules_jvm_external.zip.StableZipEntry;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.ByteStreams;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class JarUtils {
```

Comment:
was this whole file just refactored out?
(Curious if particular part needs review)

Reply:
Yes